runtime-light/stdlib/fork/fork-api.h

```diff
@@ -42,6 +42,10 @@ requires(is_optional<T>::value) task_t<T> f$wait(Optional<int64_t> fork_id_opt,
   co_return co_await f$wait<T>(fork_id_opt.has_value() ? fork_id_opt.val() : INVALID_FORK_ID, timeout);
 }
 
+inline int64_t f$get_running_fork_id() noexcept {
```

Comment:
To make it a little neater, let's visually divide code into blocks. For example, here we can easily divide it to two blocks: blocking and non-blocking API